### ISSUE TITLE
ERR_UNHANDLED_REJECTION during auth attempt

### ISSUE THREAD

**Issue by Utopiah** [valeriangalliat/firefox-sync-cli#2]
Hi, I can't login. I double checked my login/password from Firefox and even copy/pasted it to be sure but no dice :

```
fabien@fabien-CORSAIR-ONE-i160:~$ ffs auth me@domain.tld -c .ffscreds
✔ Password: … ************
node:internal/process/promises:246
          triggerUncaughtException(err, true /* fromPromise */);
          ^

[UnhandledPromiseRejection: This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason "#<Object>".] {
  code: 'ERR_UNHANDLED_REJECTION'
}
fabien@fabien-CORSAIR-ONE-i160:~$ node -v
v16.7.0
```

**Comment by valeriangalliat** [comment]
Hey! I just published version 1.0.2 with improved error handling so that you should get more information on what's going wrong, let me know if that helps :)

**Comment by Utopiah** [comment]
Thanks for the prompt reply. Indeed that solved it right away. I updated, got "'Unverified account" then check my emails and was able to connect after verifying. Thanks!
